x-pack/plugins/fleet/server/services/epm/packages/_install_package.ts

```diff
@@ -195,8 +195,8 @@ export async function _installPackage({
     const installedTemplateRefs = getAllTemplateRefs(installedTemplates);
 
     // make sure the assets are installed (or didn't error)
-    if (installKibanaAssetsError) throw installKibanaAssetsError;
     await installKibanaAssetsPromise;
+    if (installKibanaAssetsError) throw installKibanaAssetsError;
```

Comment:
Have you checked out the performance of this now? I wonder if we can just move this `await` and error handling to the call site at line 124 to be consistent with the other install functions. I'm expecting import to be faster than bulk create.

Reply:
See https://github.com/elastic/kibana/pull/119527#issuecomment-982891080. 

the asset install time is marginally better, there is very little variation in the time the asset install takes, it's always around 1 second. I am not sure that justifies any special treatment so I think you're right.

Reply:
I tested just awaiting straight away the same as the other steps. It does generally add about a second to the install time. Does the benefit of the simpler code outweigh this?